ansible/idr-centos-init.yml

```diff
@@ -4,7 +4,14 @@
 - hosts: "{{ idr_environment | default('idr') }}-hosts"
 
   tasks:
-  - name: Create persistent journald directory
+  - name: centos-init | Install locate
+    become: yes
+    yum:
+      name: mlocate
+      state: present
+    when: idr_debug | default(false) | bool
```

Comment:
Optional: If it's going to make things easier in future set a default in https://github.com/IDR/deployment/blob/IDR-0.4.7-1/ansible/group_vars/all.yml

Reply:
I thought about that, but since I haven't rolled it out elsewhere, I think I'll leave this as "internal API" for the moment. But agreed, having a section in `all.yml` with "expert" variables will be useful.

Reply:
When we move to Ansible 2.5 we can use the new `never` tag instead http://docs.ansible.com/ansible/latest/user_guide/playbooks_tags.html